src/gradualizer_lib.erl

```diff
@@ -120,6 +125,67 @@ pick_value(?type(range, [{_TagLo, _, neg_inf}, {_TagHi, _, Hi}])) ->
 pick_value(?type(range, [{_TagLo, _, Lo}, {_TagHi, _, _Hi}])) ->
     Lo.
 
+%% Prettyprints a ?type(record)
+%%
+%% What to expect:
+%% ```
+%%   #record_name{
+%%     field_one = #nested_record{
+%%       nested_field = 1
+%%     },
+%%     field_two = some_atom
+%%   }
```

Comment:
A `?type(record, [...])` is the abstract form of a *record type* but under "What to expect" is a *record value*. So, this function is prettyprinting a record type as if it were a record value?

In a record type, the fields have types but no values, e.g. `#record_name{field_one :: #nested_record{nested_field :: 1 | 5..7 | atom() }}`.

Reply:
The values that are mapped are obtained by calling `pick_value` for all the record fields.